.github/workflows/build-release-weekly.yml

```diff
@@ -70,7 +87,7 @@ jobs:
           docker run --cap-add SYS_ADMIN --device /dev/fuse \
             --security-opt apparmor:unconfined --security-opt seccomp=unconfined \
             -v /home/runner/local:/app/usr \
-            --rm mhitza/linuxdeployqt:9.5.9
+            --rm mhitza/linuxdeployqt:5.13.2
```

Comment:
from your github repo?

Reply:
Included notes about the usage of AppImage docker containers in the .github/workflows/README.md file

Reply:
As far as Bazel, I'm not sure what Bazel is and where that error shows up.